cernopendata/modules/fixtures/data/records/jade-data.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "abstract": {
+      "description": "<p>JADE real data</p>",
+      "links": [
+        {
+          "description": "All JADE runs",
+          "recid": "FIXME"
+        }
+      ]
+    },
+    "accelerator": "DESY-PETRA",
+    "collaboration": {
+      "name": "JADE collaboration",
+      "recid": "26050"
+    },
+    "collections": [
+      "JADE-Primary-Datasets"
+    ],
+    "collision_information": {
+      "energy": "12GeV",
+      "type": "e+e-"
+    },
+    "date_created": [
+      "1986"
+    ],
+    "date_published": "2024",
+    "date_reprocessed": "1986",
+    "distribution": {
+      "formats": [
+        "raw"
+      ],
+      "number_events": 7017835,
+      "number_files": 2861,
+      "size": 2614994979
+    },
+    "doi": "10.7483/OPENDATA.9S5F.BY3B",
```

Comment:
Please leave the DOI out for now. (This one would clash with an existing record.)  I'll reserve a new JADE-specific DOI when we shall be close to publishing.

Reply:
I left the field empty.